DESIGN.md

```diff
@@ -356,27 +349,15 @@ The final `subscribe` will *not* invoke `cancel`/`dispose` after receiving an `o
 
 ### JVM target and source compatibility
 
-The 2.x version will target JDK6+ but the source will be using some of the Java 8 features.
-We'll use the retrolambda project to generate JDK6 Byte-code from the Java 8 sources.
-The only Java 8 features we intend to use are: lambda expressions, method references and try-with-resources statements.
-Default methods may be considered if retrolambda support reach an acceptable level of maturity.
-
-It still up to discussion to know how we will generate multiple targets?
-Options are:
-
-- two artifcat ids: rxjava-jdk6, rxjava-jdk8
-- classifiers: jdk6, jdk8
-
-The intent is to let Android users consume the new version of RxJava.
-(Not sure if JDK6 is the appropriate requirement, JDK7+ is maybe sufficient).
+The 2.x version will target JDK6+ to let Android users consume the new version of RxJava.
 
-### Future work 
+### Future work
 
-This section contains current design work which needs more discussion and elaboration before it is merged into this document as a stated goal for 2.x. Our goal is 
+This section contains current design work which needs more discussion and elaboration before it is merged into this document as a stated goal for 2.x. Our goal is
```

Comment:
Not sure about `Our goal is`…? Next header is continuation and goal or it's not-ended sentence?